archunit/src/main/java/com/tngtech/archunit/library/dependencies/SliceCycleArchCondition.java

```diff
@@ -212,13 +212,14 @@ private ConditionEvent newEvent(Cycle<Slice, Dependency> cycle) {
         private String createDescription(Collection<String> edgeDescriptions) {
             List<String> descriptions = new ArrayList<>(edgeDescriptions);
             descriptions.add(descriptions.get(0));
-            return Joiner.on(" -> ").join(descriptions);
+            return Joiner.on(" -> " + System.lineSeparator() + "                ").join(descriptions);
         }
 
         private String createDetails(Map<String, Edge<Slice, Dependency>> descriptionsToEdges) {
             List<String> details = new ArrayList<>();
+            int count = 0;
             for (Map.Entry<String, Edge<Slice, Dependency>> edgeWithDescription : descriptionsToEdges.entrySet()) {
-                details.add(String.format("Dependencies of %s", edgeWithDescription.getKey()));
+                details.add(String.format("    %d. Dependencies of %s", ++count, edgeWithDescription.getKey()));
```

Comment:
To me it seems that an indent of 2 would achieve the same readability but waste less horizontal space :thinking: What do you think?